src/languages/1c.js

```diff
@@ -1819,10 +1819,6 @@ export default function(hljs) {
 
   return {
     name: '1C:Enterprise',
-    aliases: [
-      '1c',
-      'bsl'
```

Comment:
How did we conclude this is no longer an alias?

Reply:
I don't remember.

Reply:
I must send new commit with rollback this change?

Reply:
Yes, please.

Reply:
Return aliases